pym/bob/scm/git.py

```diff
@@ -93,7 +93,7 @@ def asScript(self):
 fi
 # checkout only if HEAD is invalid
 if ! git rev-parse --verify -q HEAD >/dev/null ; then
-    git fetch -t origin
+    git fetch -t origin +refs/heads/*:refs/remotes/origin/*
```

Comment:
This should actually be quoted with single quotes. Otherwise the shell will try to expand it as a pattern.